bin/changelog.js

```diff
@@ -19,82 +19,61 @@ const GitHubApi = require('github');
 const execSync = require('child_process').execSync;
 
 const github = new GitHubApi({ version: '3.0.0' });
+if (process.env.GITHUB_TOKEN) {
+  github.authenticate({
+    type: 'token',
+    token: process.env.GITHUB_TOKEN,
+  });
+}
 const compareCommits = RSVP.denodeify(github.repos.compareCommits);
+const getPullRequest = RSVP.denodeify(github.pullRequests.get);
```

Comment:
FWIW, we can migrate to using async/await here too (seems fine as a separate PR) and use `util.promisify` from the standard lib